tools/proofers/proof_frame_enh.inc

```diff
@@ -26,8 +36,12 @@ function echo_proof_frame_enh($ppage)
             "$code_url/scripts/text_validator.js",
             "$code_url/tools/proofers/proof_validate.js",
             "$code_url/tools/proofers/process_diacritcal_markup.js",
+            "$code_url/scripts/control_bar.js",
+            "$code_url/tools/proofers/proof_image.js",
         ],
-        "js_data" => ibe_get_js() . get_preview_js_vars() . get_preview_font_data_js() . "
+        "js_data" => get_preview_js_vars() . get_preview_font_data_js() .
+        get_control_bar_texts() . "
+        var imageData = $image_data;
         function ldAll() {
             top.initializeStuff(1);
```

Comment:
already present, but could probably use a more informative name :).

Reply:
Good idea. But unless we dig into it to discover exactly what it does any new name could be misleading.

Reply:
absolutely